src/html/manager.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Smart Tab Manager</title>
+  <meta charset=UTF-8"/>
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
+  <link rel="stylesheet" href="../css/manager.css">
+</head>
+<body>
+
+<div>
+  <input type="text" id="url">
+  <button id="btn_open_new_tab">Open!</button>
```

Comment:
Please use hyphen instead of underscore for class names and ids.
(https://google.github.io/styleguide/htmlcssguide.xml?showone=ID_and_Class_Name_Delimiters#ID_and_Class_Name_Delimiters)